src/main/scala/beam/replanning/AnchorEndTimeMutator.scala

```diff
@@ -0,0 +1,73 @@
+package beam.replanning
+
+import beam.sim.config.BeamConfig
```

Comment:
we need to enable usage of this replanning module as well

Reply:
done